docs/contributor/devenv_setup.md

```diff
@@ -60,7 +60,7 @@ ssh-copy-id -f microshift@${VMIPADDR}
 
 scp ./scripts/devenv-builder/configure-vm.sh microshift@${VMIPADDR}:
 scp ~/.pull-secret.json microshift@${VMIPADDR}:
-ssh microshift@${VMIPADDR} "~/configure-vm.sh ~/.pull-secret.json"
+ssh -t microshift@${VMIPADDR} "~/configure-vm.sh ~/.pull-secret.json"
```

Comment:
Why do we need a terminal allocation here?

Reply:
Ah, I see. Terminal is necessary when we're hitting the subscription register case.
```
+ sudo subscription-manager status
+ sudo subscription-manager register
Registering to: subscription.rhsm.redhat.com:443/subscription
Error: --username is a required parameter in non-interactive mode.
```